packages/core/data-transfer/src/file/providers/source/index.ts

```diff
@@ -72,7 +73,12 @@ class LocalFileSourceProvider implements ISourceProvider {
       // Read the metadata to ensure the file can be parsed
       this.#metadata = await this.getMetadata();
     } catch (e) {
-      throw new Error(`Can't read file "${filePath}".`);
+      if (this.options?.encryption?.enabled) {
+        throw new ProviderInitializationError(
+          `Key is incorrect or the file '${filePath}' is not a valid Strapi data file.`
```

Comment:
just a general comment, it might be worth it to provide a link to the relevant Docs.
this could help the users a lot, I think this could apply to other errors we are throwing

Reply:
I agree it could be nice to have a link to the documentation. However, I'm a bit worried about having potential broken links in the codebase over time.